dist/s_stat

```diff
@@ -66,7 +68,8 @@ txn_rts_upd_aborted_dryrun
 UNUSED_STAT_FIELDS
 
 echo "$search"
-fgrep -who "$search" $l) | sort | uniq -u > $t
+cat $l | $FGREP -wo "$search"
+} | sort | uniq -u > $t
```

Comment:
In the other scriptd, you didn't keep the `> $t`, why?

Reply:
That's because the other scripts just output the `$t` as a result but this script's output depends on the content of it.